### PR TITLE
Divide our builder container

### PR DESCRIPTION
**Pull request by cynepco3hahue**
**What this PR does / why we need it**:
Divide our builder container into two:
- the first will include all static dependencies, includes libvirt-devel
dependency
- the second will includes dynamic dependencies, some go and python
dependencies

It should reduce `virt-preview` repository issues, we still have `virt-launcher` container, but I think we can move all `virt-launcher` dependencies into `libvirt` image and it can be the second stage to stabilize our infrastructure.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by danielBelenky** [comment]
ci test please

**Comment by danielBelenky** [comment]
ci test please

File: hack/builder/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+    dnf -y install libvirt-devel-${LIBVIRT_VERSION} \
+        make \
+        git \
+        mercurial \
+        sudo \
+        gcc \
+        findutils \
+        gradle \
+        rsync-daemon \
+        rsync qemu-img \
```
[the remark this answers is not in the document]
Comment (cynepco3hahue) [reply hack/builder/Dockerfile]: true

File: hack/builder/Dockerfile [hunk 2]
```diff
@@ -13,45 +24,42 @@ RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimm
...
+    cd $GOPATH/src/k8s.io/code-generator/cmd/deepcopy-gen && \
     git checkout kubernetes-1.11.2 && \
     go install && \
-    cd /go/src/k8s.io/code-generator/cmd/defaulter-gen && \
+    # Install defaulter-gen
+    cd $GOPATH/src/k8s.io/code-generator/cmd/defaulter-gen && \
     git checkout kubernetes-1.11.2 && \
     go install && \
-    cd /go/src/k8s.io/kube-openapi/cmd/openapi-gen && \
+    cd $GOPATH && \
```
[the remark this answers is not in the document]
Comment (cynepco3hahue) [reply hack/builder/Dockerfile]: done

File: hack/builder/Dockerfile [hunk 3]
```diff
@@ -13,45 +24,42 @@ RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimm
...
+    # Install openapi-gen
+    cd $GOPATH/src/k8s.io/kube-openapi/cmd/openapi-gen && \
     git checkout c59034cc13d587f5ef4e85ca0ade0c1866ae8e1d && \
     go install && \
-    go get -u -d github.com/golang/protobuf/protoc-gen-go && \
-    cd /go/src/github.com/golang/protobuf/protoc-gen-go && \
+    cd $GOPATH && \
```
[the remark this answers is not in the document]
Comment (cynepco3hahue) [reply hack/builder/Dockerfile]: done

**Comment by cynepco3hahue** [comment]
We still has repository issues, but at least we reduced the number of such failures, and I hope when we will build images with bazel we will have zero of such issues.

**Pull request merged**